### DISCUSSION TITLE
Add Community Guidelines – X

### DISCUSSION THREAD

**Discussion by ckatzenbach** [OpenTermsArchive/terms-types#13] [Hesitations] [answered]
In the PGA meeting today (with @afisher3578 @pg-adrian @smmkolesnikov, we discussed how to handle Twitter Comm Guidelines (aka Twitter Rules) in the future. The issue is that the Twitter Rules are ver fragmented, much more than multi-page document by other platforms. Put concretely, we believe it would be important to start tracking pages such as: 
– [Crisis misinformation policy](https://help.twitter.com/en/rules-and-policies/crisis-misinformation)
[Parody, commentary, and fan account policy](https://help.twitter.com/en/rules-and-policies/parody-account-policy)
– [Coordinated harmful activity](https://help.twitter.com/en/rules-and-policies/coordinated-harmful-activity)

What's happening there is key rule-making on important aspects of digital communication on those platforms, so we should be tracking those. But how to categorise them? 

(1) Most other platforms have those types of rules within the Community Guidelines, and also Twitter has listed and linked them in the context of their Twitter Rules. So probably that would be the best high-level category. 
(2) But currently we have a finite set of sub document types for Comm Guidelines that don't encompass such policies. 

--> What shall we do?
(A) Add specific subdocument types for each of those?
(B) Add a  catch-all-remainder subdocument type/category for Comm Guidelines?
(C) Start tracking them separately in one way or the other?

**Comment by MattiSG** [comment, marked as answer]
Thanks @ckatzenbach for this question!

Regarding Community Guidelines, I believe the conclusion in the [multi-page RFC](https://github.com/OpenTermsArchive/engine/issues/773) was that there should be no named subtypes: to support exactly this case, as [eloquently](https://github.com/OpenTermsArchive/engine/issues/773#issuecomment-1089412729) brought by @pg-adrian at the time, the Community Guidelines are just one terms, made of several documents that are combined. This is apparently how you are [currently declaring](https://github.com/OpenTermsArchive/pga-declarations/blob/da4d4cbf2b9a90ba4286bbb19081581a4702fbd5/declarations/Twitter.json#L105-L183) these terms in PGA. I thus fail to understand your point (2) 🙂

If you consider that your point (1) warrants having an explicit dedicated type for some of these terms, we will welcome a [new type](https://github.com/OpenTermsArchive/terms-types/discussions/categories/new-type) proposal!

**Comment by ckatzenbach** [comment]
Alright, then that is good news. Our impression was that [this definition of term types (including subtypes for Community Guidelines)](https://github.com/OpenTermsArchive/terms-types/blob/main/termsTypes.json) means that there should be finite list. I believe if we can keep this flexibility, it makes sense to not structure this too much and just add them to the declaration file for Twitter.

**Reply by MattiSG** [reply]
Thanks @ckatzenbach for explaining the reasoning!
I am sorry for these subtypes, they definitely are misleading! They are a remnant from https://github.com/OpenTermsArchive/engine/issues/780, and should have been cleaned in https://github.com/OpenTermsArchive/engine/pull/891. We will do that when we can (contributions welcome!).
